src/backend/Functions/ProcessWorkflowQueue.cs

```diff
@@ -14,17 +15,43 @@ public static class ProcessWorkflowQueue
         [FunctionName("ProcessWorkflowQueue")]
         public static async Task Run([QueueTrigger("apiportworkflowqueue")]WorkflowQueueMessage workflowMessage,
             [Queue("apiportworkflowqueue")]ICollector<WorkflowQueueMessage> workflowMessageQueue,
-            ILogger log)
+            ILogger log,
+            CancellationToken cancelToken)
         {
-            log.LogInformation("Processing message {SubmissionId}, stage {Stage}", workflowMessage.SubmissionId, workflowMessage.Stage);
+            Action<WorkflowQueueMessage> cancelAsync = (msg) =>
+            {
+                log.LogError("Timeout during workflow action execution.SubmissionId: { msg.SubmissionId}, Stage: { msg.Stage}", workflowMessage.SubmissionId, workflowMessage.Stage);
```

Comment:
could condense this into:
```cs
var errorMessage = "Timeout during workflow action execution.SubmissionId: {msg.SubmissionId}, Stage: {msg.Stage}";

log.LogError(errorMessage, workflowMessage.SubmissionId, workflowMessage.Stage);
throw new TimeoutException(errorMessage); 

```

Reply:
ILogger is using structured logging, so the strings are actually different.

Reply:
Yes, leaving as-is for reason that taylor mentioned.

Reply:
Good to know! Now I know. :D